backend/app/main.py

```
@@ -139,6 +142,32 @@ async def lifespan(app: FastAPI):
         # UPSTAGE_API_KEY 미설정·네트워크 오류·모델 로드 실패 등 — 서버 기동은 계속
         _log.warning(f"공익직불 RAG 준비 실패 (/subsidy/ask 제한 동작): {e}")
 
+    # 💡 3일 지난 이미지를 삭제하는 백그라운드 태스크 시작
+    async def cleanup_old_diagnosis_images():
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `cleanup_old_diagnosis_images`

Add return type annotation: `None`

(ANN202)